migrations/906-abuse-report-read.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `abuse_reports`
+    ADD COLUMN `read` bool NOT NULL,
```

Comment:
No. But you could default to false in model and db and add an `UPDATE` for the data that's already there ?